portlet-parent/wicketstuff-portlet/src/main/java/org/apache/wicket/portlet/WicketPortlet.java

```diff
@@ -510,7 +494,7 @@ else if (redirectLocation != null) {
 				}
 				else {
 					if (LOG.isDebugEnabled()) {
-						LOG.debug("ajax redirect url after inclusion:" + redirectLocation);
+						LOG.debug("ajax redirect url after inclusion:" + responseState.getAjaxRedirectLocation());

```

Comment:
I would use `LOG.debug("ajax redirect url after inclusion: {}", responseState.getAjaxRedirectLocation());` here

Reply:
Will do this. 
Just waiting for the discussion below, to change them all together.